lib/mutant/cli.rb

```diff
@@ -207,7 +207,7 @@ def update(attributes)
     #
     # @api private
     def add(attribute, value)
-      update(attribute => config.public_send(attribute).dup << value)
+      with(attribute => config.public_send(attribute).dup << value)
```

Comment:
You are correct. This code predates my preference for binary, non destructive operations by default. Even when the frozen values do not leak the context.

Will fix that.